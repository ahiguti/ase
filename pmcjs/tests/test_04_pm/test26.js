
use('Perl');
Perl.Use("MIME::QuotedPrint");
var qp = Perl.Functions["MIME::QuotedPrint"];
var s1 = "abc=def";
var s2 = qp.encode_qp(s1);
var s3 = qp.decode_qp(s2);
print(s1);
print(s2);
print(s3);

