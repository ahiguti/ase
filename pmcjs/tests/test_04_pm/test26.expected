abc=def
abc=3Ddef=

abc=def
